test-packages/e2e-tests/test/rest.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  TestEntity,
+  TestServiceApiRequestBuilder
+} from '@sap-cloud-sdk/test-services/rest/test-service';
+import { destination } from './test-util';
+
+// TODO: How do I handle paths in rest requests?
+// TODO: Transpilation needed + tsconfig needs dom typings
+// TODO: Response is never "parsed"
```

Comment:
1 and 3 are open for discussion. If you have opinions leave a comment.

Reply:
1. Do you mean how to you test the generated url in the request or custom path or custom path parameter?
2. Discussed in the planning
3. As a minimal scope, I would like to see at least one required property (line 22 `keyProperty`) and one optional property (line 23 `stringProperty`) are tested. Is there any technical difficulty to parse the response?

Reply:
1. We currently don't have a `withCustomServicePath` function here. That is why I have to touch the destination to get it from the correct url. I guess we should allow to set that.
3. The requestBuilder currently returns an AxiosResponse. I think we should rather return the data directly on execute, similar to how we do it for OData. We could then have a second executeRaw function that also returns the response (as discussed for OData).

What do you think about that?

As to the minimal scope of the test - do you think there is a case missing? Testing that there are required properties and optional properties should probably be part of a type test. Because this is covered by the type (in this case `TestEntity`) right?